src/python/WMCore/MicroService/MSPileup/MSPileup.py

```diff
@@ -106,7 +94,8 @@ def createPileup(self, pdict):
         :return: results of MSPileup data layer (list of dicts)
         """
         authz_match(self.role, self.group)
-        return self.mgr.createPileup(pdict)
+        # TODO: Implement logic for creating a pileup object and persist in the db
```

Comment:
I'm not sure it is right decision in this PR. I suggest to not touch the logic here since it is correct delegation of creating the pileup object, i.e. the data layer should create it via its `createPileup` API. Or, at least we should discuss it and make such change in separate PR. This PR should only fix `mspileupError` loading issue and not more.